### DISCUSSION TITLE
How to merge files in Windows.

### DISCUSSION THREAD

**Discussion by JavierLinaresCorporativo** [mikefarah/yq#1570] [Q&A]
Hi,

I'm new in yq and after download yq_windows_amd64.zip I can't use 'm' flag like example: https://mikefarah.gitbook.io/yq/v/v2.x/merge

For example, I can update de file inplace with this command:
`yq_windows_amd64 e -i m swagger.yaml`

I can't use 'm' flag because throws error:
`yq_windows_amd64 m swagger.yaml
Error: 1:1: invalid input text "m"`

¿What is the correct sintax for windows executable?

Thanks.

**Comment by JavierLinaresCorporativo** [comment]
Oh, I've read 2.x doc and use 4.31.1 exe. I've downloaded v2 and use merge command instead. Sucess!

**Reply by mikefarah** [reply]
Glad it worked for you - however 2.0 is quite old and no longer supported. There will be security vulnerabilities in the compiler it was built with as well as the dependencies. 

I highly recommend switching to 4.x and ensure you've got latest, most secure version. Merge is a little more complex (but more powerful) in v4 - it's documented here: https://mikefarah.gitbook.io/yq/operators/multiply-merge

**Reply by JavierLinaresCorporativo** [reply]
> Glad it worked for you - however 2.0 is quite old and no longer supported. There will be security vulnerabilities in the compiler it was built with as well as the dependencies.
> 
> I highly recommend switching to 4.x and ensure you've got latest, most secure version. Merge is a little more complex (but more powerful) in v4 - it's documented here: https://mikefarah.gitbook.io/yq/operators/multiply-merge

Thanks for your answer, if I need again, I will use your solution.
